### PR TITLE
[tools/onert_train] Change default learning rate

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit changes the default learning rate value to 0.001.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Comment by jyoungyun** [comment]
The value `0.001` is not an absolute value. However, we usually compare the result of tensorflow, and tensorflow sets the 0.001 as the default learning rate. So it changes the default value to make it easier to test.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM
